__tests__/app.test.js

```diff
@@ -339,3 +339,45 @@ describe("PATCH api/reviews/:review", () => {
       });
   });
 });
+describe("GET api/reviews/(queries)", () => {
+  test("returns", () => {
+    return request(app)
+      .get("/api/reviews?category=social deduction&sort_by=review_id&order=asc")
+      .expect(200)
+      .then(({ body }) => {
+        const { reviews } = body;
+        reviews.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              owner: expect.any(String),
+              title: expect.any(String),
+              review_id: expect.any(Number),
+              category: "social deduction",
+              review_img_url: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              designer: expect.any(String),
+              comment_count: expect.any(Number),
+            })
+          );
+        });
+      });
+  });
```

Comment:
Good job on the tests, we just need to do some extra checks for a few more behaviours, will write them down so you can refer to them for ideas 🙂 
- invalid sort_by i.e. sort_by=banana
- invalid order i.e. order=banana
- invalid category i.e. category=banana